intro.md

```diff
@@ -144,87 +193,70 @@ Note: kafka sources/ sinks can also reside outside of DSH
+![dsh-overview-2](images/dsh/dsh-sourcessinks.svg)<!-- .element: class="stretch" style="background:none; border:none; box-shadow:none;" width="100%" -->
 
 <!--v-->
-## MQTT bridge
-![dsh-overview-2](images/dsh/dsh-kafkamqtt.svg)<!-- .element: class="stretch" style="background:none; border:none; box-shadow:none;" width="30%" -->
-- Protocol adapter
-    - MQTT interface with Kafka
-- Like MQTT: allows wildcard subscriptions:<!-- .element: class="fragment" data-fragment-index="2" -->
-
- ```
- /platform/stream/topic/#
- ```
- <!-- .element: class="fragment" data-fragment-index="2" -->
-
-Note: wildcard subscriptions later, in the MQTT part
+<!-- .slide: data-transition="fade" -->
+## Overview
+![dsh-overview-2](images/dsh/dsh-sourcessinks-singlestream.svg)<!-- .element: class="stretch" style="background:none; border:none; box-shadow:none;" width="100%" -->
+Note: access to this stream has two sides; from outside, and from inside (the DSH). First, we'll deal with the 'inside'. 
+Let's say the data source (sensor) is _point at member of audience_ yours. That makes the data, and the stream, yours as well.
 
 <!--v-->
-## External data sources 
-
-- are not always MQTT ... <!-- .element: class="fragment" data-fragment-index="2" -->
-<!-- - do not always stream -->
-- ... and will therefore require custom adapters  <!-- .element: class="fragment" data-fragment-index="3" -->
-
-We allow tenants to write their own adapters  <!-- .element: class="fragment" data-fragment-index="4" -->
-
-Note: do we say other adapters are in the works?
-
-<!--s-->
 <!-- .slide: data-transition="fade" -->
-
-# Kafka
-
-![dsh-overview-2](images/dsh/dsh-kafkamqtt.svg)<!-- .element: class="stretch" style="background:none; border:none; box-shadow:none;" width="100%" -->
+## Overview
+![dsh-overview-2](images/dsh/dsh-sourcessinks-singletenant.svg)<!-- .element: class="stretch" style="background:none; border:none; box-shadow:none;" width="100%" -->
+Note: _point at member of audience_ this is your tenant
 
 <!--v-->
-
-## Kafka
-Three Kafka stream-types
--  _stream._ topic
--  _internal._ topic 
--  _scratch._ topic
+<!-- .slide: data-transition="fade" -->
+## Overview
+![dsh-overview-2](images/dsh/dsh-sourcessinks-singletenant-stream.svg)<!-- .element: class="stretch" style="background:none; border:none; box-shadow:none;" width="100%" -->
+Note: within your tenant, you run your stuff on your stream.
 
 <!--v-->
 <!-- .slide: data-transition="fade" -->
-![kafkastreams](images/dsh/dsh-kafkastreams.svg)<!-- .element: class="stretch" style="background:none; border:none; box-shadow:none;" width="100%" -->
+## Overview
+![dsh-overview-2](images/dsh/dsh-sourcessinks-scratch.svg)<!-- .element: class="stretch" style="background:none; border:none; box-shadow:none;" width="100%" -->
+Note: if you have multiple things running in your tenant, you can make use of a stream type that is not available outside your tenant. You cannot share this stream.
```

Comment:
Recurring theme; please avoid 'stuff', 'things', etc. In this case you'd be running multiple _services_ in your tenant.
You make use of a 'topic' type, not a 'stream' type. A 'stream' is a type of 'topic'.

Reply:
Agreed on the stuff (ha), and fix'd. Terminology is still an issue, which I'd like to revisit (and then put in another commit).